showcase/src/data/data.js

```diff
@@ -0,0 +1,31 @@
+export default [
+  {
+    image: 'http://via.placeholder.com/180x320',
+    username: 'test user',
```

Comment:
Put the username first and maybe rename this to be displayName as I don't usually think of usernames as allowing spaces.

Reply:
Or both a username and display name?